configs/DAKEFPVH743/config.h

```diff
@@ -0,0 +1,152 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        DAKEFPVH743
+#define MANUFACTURER_ID   DAKE
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+#define BEEPER_PIN           PE2
+#define MOTOR1_PIN           PA0
+#define MOTOR2_PIN           PA1
+#define MOTOR3_PIN           PA2
+#define MOTOR4_PIN           PA3
+#define MOTOR5_PIN           PE9
+#define MOTOR6_PIN           PE11
+#define MOTOR7_PIN           PE13
+#define MOTOR8_PIN           PE14
+#define LED_STRIP_PIN        PB0
+#define SERVO1_PIN           PC9  // S9
+#define SERVO2_PIN           PC8  // S10
+#define SERVO3_PIN           PD15 // S11
+#define SERVO4_PIN           PD14 // S12
+#define UART1_TX_PIN         PA9
+#define UART1_RX_PIN         PA10
+#define UART2_TX_PIN         PD5
+#define UART2_RX_PIN         PD6
+#define UART3_TX_PIN         PD8
+#define UART3_RX_PIN         PD9
+#define UART4_TX_PIN         PD1
+#define UART4_RX_PIN         PD0
+#define UART5_TX_PIN         PB6
+#define UART5_RX_PIN         PB5
+#define UART6_TX_PIN         PC7
+#define UART6_RX_PIN         PC8
+#define UART7_TX_PIN         PE8
+#define UART7_RX_PIN         PE7
+#define UART8_TX_PIN         PE1
+#define UART8_RX_PIN         PE0
+#define I2C1_SCL_PIN         PB10
+#define I2C1_SDA_PIN         PB11
+#define LED0_PIN             PD10
+#define LED1_PIN             PD11
+#define LED2_PIN             PD12
+#define SPI1_SCK_PIN         PA5
+#define SPI1_SDI_PIN         PA6
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SCK_PIN         PB13
+#define SPI2_SDI_PIN         PB14
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SCK_PIN         PC10
+#define SPI3_SDI_PIN         PC11
+#define SPI3_SDO_PIN         PC12
+#define CAMERA_CONTROL_PIN   PB8
+#define ADC_VBAT_PIN         PC1
+#define ADC_RSSI_PIN         PC5
+#define ADC_CURR_PIN         PC0
+#define PINIO1_PIN           PE3
+#define PINIO2_PIN           PE4
+#define PINIO3_PIN           PE5
+#define PINIO4_PIN           PE6
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_EXTI_PIN      PE15
+#define GYRO_2_CS_PIN        PE11
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0,  PA0,  1,  0) \
+    TIMER_PIN_MAP( 1,  PA1,  1,  1) \
+    TIMER_PIN_MAP( 2,  PA2,  1,  2) \
+    TIMER_PIN_MAP( 3,  PA3,  1,  3) \
+    TIMER_PIN_MAP( 4,  PE9,  1,  4) \
+    TIMER_PIN_MAP( 5,  PE11, 1,  5) \
```

Comment:
This seems wrong

Reply:
![image](https://github.com/user-attachments/assets/e442d656-c372-46ec-81a3-faf8cb7b27b8)
It seems like that's it here, it's all `AF1`.